### PR TITLE
add miktex 22.3

### PR DESCRIPTION
**Pull request by gratuxri**
- [ X ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ X ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ X ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ X ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ X ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

-----
Add new Upstream Version

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/55090)

### CODE REVIEW

File: manifests/c/ChristianSchenk/MiKTeX/22.3/ChristianSchenk.MiKTeX.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- interactive
+- silentWithProgress
+InstallerSwitches:
+  Silent: --unattended
+  SilentWithProgress: --unattended
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://mirror.kumi.systems/ctan/systems/win32/miktex/setup/windows-x64/basic-miktex-22.3-x64.exe
+  InstallerSha256:
```
Comment (Trenly) [inline manifests/c/ChristianSchenk/MiKTeX/22.3/ChristianSchenk.MiKTeX.installer.yaml:19]:
```suggestion
  InstallerSha256: 17D1CCAD25E7182B07872C11D3440F66281EA64782C4CD6FA0850EF75D3AAA59
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by gratuxri** [comment]
Thanks, I have forget it to commit.

File: manifests/c/ChristianSchenk/MiKTeX/22.3/ChristianSchenk.MiKTeX.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+InstallModes:
+- interactive
+- silentWithProgress
+InstallerSwitches:
+  Silent: --unattended
+  SilentWithProgress: --unattended
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://mirror.kumi.systems/ctan/systems/win32/miktex/setup/windows-x64/basic-miktex-22.3-x64.exe
```
Comment (ItzLevvie) [inline manifests/c/ChristianSchenk/MiKTeX/22.3/ChristianSchenk.MiKTeX.installer.yaml:18]:
```suggestion
  InstallerUrl: https://miktex.org/download/ctan/systems/win32/miktex/setup/windows-x64/basic-miktex-22.3-x64.exe
```
If that mirror is unavailable then the URL will not work.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/c/ChristianSchenk/MiKTeX/22.3/ChristianSchenk.MiKTeX.installer.yaml (hunk 2 above)
Comment (gratuxri) [reply manifests/c/ChristianSchenk/MiKTeX/22.3/ChristianSchenk.MiKTeX.installer.yaml:18]: Are redirects for CDNs not supported?

**Review by ItzLevvie — APPROVED** [review]

**Comment by mboom** [comment]
It seems this package installer is no longer available on miktex.org. Please update.

**Comment by gratuxri** [comment]
I don't have windows here right now, can you please validate #69100 and install it locally?

**Comment by mboom** [comment]
The validate command failed. It seems manifest version value 1.2.0 is not supported.
> winget validate --manifest .
Manifest validation failed.
Manifest Error: The multi file manifest has inconsistent field values. Field: ManifestVersion Value: 1.2.0 File: ChristianSchenk.MiKTeX.locale.en-US.yaml
Manifest Error: The multi file manifest has inconsistent field values. Field: ManifestVersion Value: 1.2.0 File: ChristianSchenk.MiKTeX.yaml


I changed the manifest value in all three files to 1.1.0 and then the validation works.
> winget validate --manifest .
Manifest validation succeeded.

**Comment by gratuxri** [comment]
Thank you, I have pushed new changes for #69100 and waiting now for review and merge. I have found new PR with the same solution: #69103

**Comment by mboom** [comment]
Thank you! I can confirm that the installation completed successfully. I'm glad that this issue is resolved.

**Pull request merged**